ui/raidboss/data/02-arr/dungeon/brayfloxs_longstop.ts

```diff
@@ -67,12 +78,14 @@ export default {
       // The pack with the boss is 3 Violet Backs, not parsing for them prevents the trigger
       // from activating early when you pick up the Headgate Key and the boss and adds spawn.
       id: 'Brayflox Normal Pelican Adds',
+      type: 'AddedCombatant',
```

Comment:
Confirming assumptions if addedCombatantFull should be type AddedCombatant

Reply:
Yes. Functionally they're the same behind the scenes as far as return fields, so it makes no sense to split them into separate types.

Same for GameLog below.

If the typing wasn't correct, this would error in tsc.

However, given that this trigger doesn't use data/matches/output directly (it's all done with canned helpers) you should be able to leave the type out?